tests/units/Model/UserModelTest.php

```diff
@@ -392,6 +392,6 @@ public function testDisablePrivateProjects()
         $this->assertTrue($userModel->disable(2));
 
         $project = $projectModel->getById(2);
```

Comment:
This test was incorrect before your changes. Since we want to verify if a personal project is disabled when a user is disabled, it should be `$projectModel->getById(1)`. And the assertion below should be the same as before.

Reply:
There was also another mistake in this test, the ower_id was 0 every time, therefore the project was not disabled, because it was not owned by user 2. Should be fine now.